frontend/src/concepts/pipelines/content/pipelinesDetails/pipeline/PipelineDetails.tsx

```diff
@@ -65,9 +82,12 @@ const PipelineDetails: PipelineCoreDetailsPageComponent = ({ breadcrumbPath }) =
               }
               empty={false}
               loaded={pipelineLoad && pipelineTemplateLoaded}
-              loadError={pipelineLoadError || templateLoadError}
+              loadError={templateLoadError}
               headerAction={
-                <PipelineDetailsActions onDelete={() => setDeleting(true)} pipeline={pipeline} />
+                pipelineLoad &&
+                pipelineTemplateLoaded && (
+                  <PipelineDetailsActions onDelete={() => setDeleting(true)} pipeline={pipeline} />
+                )
```

Comment:
Curious. Why not take the entire error up to the other block? Is there a reason we would want to show some details and not all 🤔 

Also why the change to the `headerAction` here? What state are you trying to fix here?

Reply:
Because the templateLoad Error shows different error ( here the pipeline is present but the format or yaml file is broken)
![withActionButton](https://github.com/opendatahub-io/odh-dashboard/assets/99238909/d8b138ff-8a9e-4b27-832e-f869b7fb9a49)